front/src/common/Map/Layers/Signals.tsx

```diff
@@ -81,9 +81,9 @@ function Signals(props: PlatformProps) {
           (signalAspect) =>
             signalAspect.signal_id === renderedDynamicStopsFeature.id &&
             !isNil(signalAspect.time_start) &&
-            signalAspect.time_start <= new Date(timePosition) &&
+            signalAspect.time_start <= new Date(timePosition as string) &&
```

Comment:
```suggestion
            signalAspect.time_start <= new Date(timePosition) &&
```
No need for "as string" as timeString is a string now

Reply:
yes